src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -153,21 +162,35 @@ namespace Uno.UI {
 				Lottie.raiseState(animation);
 			});
 
-			if (animation.isLoaded) {
+			(animation as any).addEventListener("loopComplete", (e: any) => {
 				Lottie.raiseState(animation);
-			} else {
-				(animation as any).addEventListener("data_ready", (e: any) => {
-					Lottie.raiseState(animation);
-				});
-			}
+			});
+
+			(animation as any).addEventListener("segmentStart", (e: any) => {
+				Lottie.raiseState(animation);
+			});
+
+			(animation as any).addEventListener("data_ready", (e: any) => {
+				Lottie.raiseState(animation);
+			});
+
+			Lottie.raiseState(animation);
 
 			return runningAnimation;
 		}
 
+		private static getStateString(animation: Lottie.AnimationItem): string {
+			const duration = animation.getDuration(false);
+
+			const state = `${animation.animationData.w}|${animation.animationData.h}|${animation.isLoaded}|${animation.isPaused}|${duration}`;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)